### PR TITLE
added Polish translation

### PR DESCRIPTION
**Pull request by tusindfryd**

### CODE REVIEW

**Comment by DeeDeeG** [comment]
Hello @tusindfryd,

Thank you for the translation! We will look for another native speaker of Polish to review this, and then we can accept/merge.

**Comment by DeeDeeG** [comment]
It may take us some time to find a reviewer, as most of the team has been busy with other responsibilities lately. If we do have any updates for you, we'll post them here. Thanks again for the contribution.

**Comment by cllns** [comment]
@katafrakt Any chance you could take a look at these translations? :) 

Here are the English versions: https://github.com/RefugeRestrooms/refugerestrooms/tree/develop/config/locales/en

File: config/locales/pl/about.pl.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+pl:
+  about:
+    title: 'O REFUGE'
+    p1header: 'Czym jest REFUGE Restrooms?'
+    p1:
+      first: 'Refuge Restrooms to aplikacja u�atwiaj�ca osobom transp�ciowym i interp�ciowym bezpieczny dost�p do �azienek publicznych. U�ytkownicy mog� znale�� �azienki znajduj�ce si� w promieniu konkretnej lokalizacji, doda� now� �azienk� do listy, a tak�e skomentowa� i oceni� te dodane ju� wcze�niej.'
```
Comment (katafrakt) [inline config/locales/pl/about.pl.yml]: "w pobliżu konkretnego miejsca" (promień wymagałby podania liczby, np. w promieniu 5 km od punktu X; lokalizacja jest mało polskim słowem, właściwie kalką językową)

File: config/locales/pl/about.pl.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+      second: 'Naszym celem jest nie tylko znajdowanie obecnie istniej�cych ju� �azienek bezpiecznych dla os�b transp�ciowych, ale tak�e propagowanie dzia�a� maj�cych na celu zwi�kszenie bezpiecze�stwa os�b transp�ciowych i interp�ciowych.'
+    p2header: 'Sk�d pochodz� wszystkie dane?'
+    p2: 'Pierwsze 4500 lokalizacji pochodzi z danych nieprowadzonego ju� projektu Safe2Pee. Pozosta�e s� dodawane przez u�ytkownik�w. Je�eli zauwa�ysz, �e �azienka w miejscu publicznym jest neutralna, dodaj j� do bazy danych.'
+    p3header: 'Sk�d nazwa REFUGE (ang. azyl)?'
+    p3: 'Chcemy, �eby nasi u�ytkownicy byli godnie traktowani: bezpieczny dost�p do �azienki do podstawowe prawo cz�owieka. Chcemy zapewni� to bezpiecze�stwo ka�demu w razie potrzeby.'
```
Comment (katafrakt) [inline config/locales/pl/about.pl.yml]: Może zostawić tytuł "Skąd nazwa REFUGE?" i tłumaczenie dodać do `p3`? To tylko sugestia stylistyczna.

File: config/locales/pl/contacts.pl.yml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+pl:
+  contacts:
+    new:
+      request-edit-for-restroom: 'Zasugeruj zmian� w �azience %{restroom_name}'
```
Comment (katafrakt) [inline config/locales/pl/contacts.pl.yml]: Polskie znaki się zepsuły chyba tutaj i niżej w tym pliku.

File: config/locales/pl/footer.pl.yml [hunk 4]
```diff
@@ -0,0 +1,15 @@
+pl:
+  layouts:
+    footer:
+      aria-labels:
+        github: 'Refuge Restrooms na GitHubie'
+        twitter: 'Refuge Restrooms na twitterze'
```
Comment (katafrakt) [inline config/locales/pl/footer.pl.yml]: Twitter raczej wielką literą

Comment (katafrakt) [reply config/locales/pl/footer.pl.yml]: Chociaż widzę że w angielskim oryginale jest małą (podobnie jak Tumblr poniżej)

File: config/locales/pl/navigation.pl.yml [hunk 5]
```diff
@@ -0,0 +1,10 @@
+pl:
+  layouts:
+    navigation:
+      toggle-navigation-button-label: 'Nawigacja'
+      submit-a-new-restroom-hyperlink-label: 'Dodaj �azienk�'
```
Comment (katafrakt) [inline config/locales/pl/navigation.pl.yml]: tu również problem z polskimi znakami

**Comment by katafrakt** [comment]
And one more thing: does it make sense to translate `text_msg.en.yml`? It's an American and there is no mention that texts should be in English. Maybe it's better to tell users it's not available in their language.

**Comment by tusindfryd** [comment]
Yeah I messed up the encoding at first. Fixed it now, though. Thanks for the stylistic suggestions :)

File: config/locales/pl/signs.pl.yml [hunk 6]
```diff
@@ -0,0 +1,8 @@
+pl:
+  pages:
+    signs:
+      title: 'Pobierz znaki do łazienek'
+      intro: 'Jeżeli w Twojej placówce znajduje się publiczna łazienka dostępna niezależnie od płci, możesz dodać ją do bazy danych i udostępnić któryś z poniższych znaków. W ten sposób dasz znać odwiedzającym, że cenisz ich bezpieczeñstwo i komfort. Pliki są w formacie PDF, możliwe do wydrukowania w wysokiej rozdzielczości.'
```
Comment (katafrakt) [inline config/locales/pl/signs.pl.yml]: Also here in "bezpieczeństwo"

**Comment by DeeDeeG** [comment]
Hi again,

Thank you @tusindfryd for making updates/changes, and to @katafrakt for the review! Much appreciated!

**Comment by DeeDeeG** [comment]
I'm going to answer some questions that were raised in comments above.

> Why are Twitter and Facebook  in `contacts.en.yml` without capital letter?

I think we have "twitter" in lower-case in `footer.en.yml`, but everything in `contacts.en.yml` looks properly capitalized, unless I missed something. Twitter is un-capitalized for no particular reason, and perhaps we should change it to "Twitter" with capital "T". (I always thought Twitter themselves styled it "twitter," but I see on their website that they use capital "T." 🤷)

> . . . does it make sense to translate `text_msg.en.yml`? It's an American and there is no mention that texts should be in English. Maybe it's better to tell users it's not available in their language.

I think this service will work no-matter where you text from? (Maybe there are fees associated with international texting.) I think a warning that texts from this service are in English is a fine idea, though. And maybe a warning that you may be charged texting fees. In which case, a warning regarding language/fees should probably be added to the other locales as well.

This page is not linked to from anywhere, so no-one sees it at the moment. Users would need to know to visit this URL: https://refugerestrooms.org/text

I think we should modify the nav bar at the top to link to the text service page, though, so keeping the translation up-to-date is a good idea IMO. And I think changing the text to say it is "unavailable in Poland" is perhaps a more-agressive warning than needed. Open to discussion and feedback from anyone on this. Thanks again.

File: config/locales/pl/text_msg.pl.yml [hunk 7]
```diff
@@ -0,0 +1,5 @@
+pl:
+  pages:
+     text:
+        title: 'Usługa niedostęna w Polsce.'
```
Comment (DeeDeeG) [inline config/locales/pl/text_msg.pl.yml]: This PR is already merged, but I forgot to address something.

I believe the SMS feature *can* be used outside of the U.S. (I haven't tested this personally, but I see no reason why it wouldn't. Again, there might be  international textig fees, if your cell service provider charges for that, but it should still work... There may not be many restrooms in Poland yet, but hopefully there willl be some eventually.)

(Also, the responses are simply an address. See: https://github.com/RefugeRestrooms/refugerestrooms/blob/develop/app/assets/images/text-msg-demo.png)

Could I possibly ask you to submit another PR reverting commit 96ec91e, so the Polish translation for this file goes back to being a full translation (not warning users the service won't work outside the U.S.)?

Or would you mind if I just go ahead and revert that commit in our `develop` branch?

Thanks, and sorry for the oversight in merging before resolving that issue.

Comment (DeeDeeG) [reply config/locales/pl/text_msg.pl.yml]: Actually, As far as I can tell the SMS service does not work at the moment.

I tried texting the number, and it seems to be in use by Lyft. See this issue for details: https://github.com/RefugeRestrooms/refugerestrooms/issues/476

For now this PR is fine as-is. Thanks again for this.

**Pull request merged**